source/fuzz/transformation_permute_function_parameters.cpp

```diff
@@ -91,18 +91,17 @@ void TransformationPermuteFunctionParameters::Apply(
       fuzzerutil::GetFunctionType(ir_context, function);
   assert(old_function_type_inst && "Function must have a valid type");
 
-  std::vector<uint32_t> type_ids = {
-      old_function_type_inst->GetSingleWordInOperand(0)};
+  std::vector<uint32_t> type_ids;
```

Comment:
If you're gonna make other changes, could call this "parametr_type_ids" to be even clearer.

Reply:
Fixed in 5bf3872.